src/main/java/no/ssb/guardian/maskinporten/config/MaskinportenClientConfig.java

```diff
@@ -1,31 +1,40 @@
-package no.ssb.maskinporten.guardian.config;
+package no.ssb.guardian.maskinporten.config;
 
```

Comment:
import no.ssb.guardian.maskinporten.util.ConfigValidator;

package no.ssb.guardian.maskinporten.util;


import java.util.regex.Pattern;

public class ConfigValidator {

    public static final String INVALID_AUDIENCE_MESSAGE = "Audiance must be ended with a backslash (/)";
    public static final String AUDIENCE_VALIDATOR = ".*([.,\/])"; // better regex pattern?
    private static final Pattern AUDIENCE_PATTERN = Pattern.compile(AUDIENCE_VALIDATOR);

    public static boolean isValidAudience(String name) {
        return AUDIENCE_PATTERN.matcher(name).matches();
    }
}